operations/experimental/add-ipv6-security-group.yml

```diff
@@ -0,0 +1,22 @@
+# --- Add IPv6 Security Groups ---
+- type: replace
```

Comment:
You can use the "-" syntax also here:
```
- type: replace
  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/default_running_security_groups/-
  value: ipv6_public_networks
```
This adds the "ipv6_public_networks" at the end of the array. So you don't need to repeat the existing values and it interferes less with possible other custom ops files.